indigo_app/templates/indigo_api/_task_detail.html

```diff
@@ -116,6 +116,31 @@ <h5>
 
                 <div class="text-right">
                   <input class="btn btn-primary" type="submit" value="Add comment" id="id_submit" disabled>
+                  
+                  {% if task.state == 'pending_review' %}
+                    {% if task.unsubmit_task_permission %}
+                      <input
+                        class="btn btn-primary mr-2 float-left" 
+                        type="submit" 
+                        data-confirm="Are you sure you want to comment and request changes to this task?" 
+                        formaction="{% url 'unsubmit_task' place=place.place_code pk=task.pk %}" 
+                        id="comment_request_changes"
+                        value="Request changes" 
+                      />
+                    {% endif %}
+                    {% if task.close_task_permission %}
+                      <input 
+                        class="btn btn-success mr-2 float-left"
+                        type="submit"
+                        data-confirm="Are you sure you want to approve this task?" 
+                        formaction="{% url 'close_task' place=place.place_code pk=task.pk %}" 
+                        id="comment_approve" 
+                        value="Approve"
+                      />
+                    {% elif task.customised.close_url %}
```

Comment:
This needs to be the start of the if statement -- we only show the approve button if the task does NOT have a customised closed url.